docs/HowToChooseTuner.md

```diff
@@ -230,11 +230,33 @@ _Usage_:
       start_step: 5
 ```
 
+<a name="Curvefitting"></a>
+**Curvefitting**
+
+Curve Fitting Assessor is a LPA(learning, predicting, assessing) algorithm. It stops a pending trial X at step S if the prediction of final epoch's performance worse than the best final performance in the trial history. In this algorithm, we use 12 curves to fit the accuracy curve, the large set of parametric curve models are chosen from [reference paper][9]. The learning curves' shape coincides with our prior knowlwdge about the form of learning curves: They are typically increasing, saturating functions.
+
+_Suggested scenario_: It is applicable in a wide range of performance curves, thus, can be used in various scenarios to speed up the tuning progress. Even better, it's able to handle and assess curves with similar performance.
+
+_Usage_:
+```yml
+  assessor:
+    builtinAssessorName: Curvefitting
+    classArgs:
+      # (optional) A trial is determined to be stopped or not, 
+      * only after receiving start_step number of reported intermediate results.
+      * The default value of start_step is 6.
+      start_step: 6
+      # The total number of epoch
+      * The default value of start_step is 20.

```

Comment:
start_step? And what is epoch_num used for?

Reply:
In order to save our computing resource, we start to predict when we have more than start_step(default=6) accuracy points. Start_step is an optional choice.
We need to know the number of epoch to determine which point we need to predict, since the final value corresponds to the accuracy of last epoch.